commands/command_status.go

```diff
@@ -31,6 +33,9 @@ func statusCommand(cmd *cobra.Command, args []string) {
 	if porcelain {
 		porcelainStagedPointers(scanIndexAt)
 		return
+	} else if status_json {
```

Comment:
yeah that's fair, I kept it as `if ... else if` so it's more obvious that they are disparate branches at a glance (without having to read the code inside the blocks)